src/plugins/recurring-issues/index.js

```diff
@@ -15,9 +15,40 @@ const moment = require("moment-timezone");
 async function getReleaseIssueBody(releaseDate) {
     return `
 
-The scheduled release on ${releaseDate.format("dddd, MMMM Do, YYYY")} is assigned to:
+The next scheduled release will occur on ${releaseDate.format("dddd, MMMM Do, YYYY")}.
 
-* (needs volunteer)
+## Release Day Checklist
+
+- [ ] Remove the 'tsc agenda' label on this issue
+- [ ] Review open pull requests and merge any that are ready
```

Comment:
The [old instructions](https://eslint.org/docs/maintainer-guide/releases) include a few lines about what's "ready", but that's already defined in our maintainer guide and doesn't need redundant redefinition, so what do you think of linking to the maintainer guide here?

```suggestion
- [ ] Review open pull requests and merge any that are [ready](https://eslint.org/docs/maintainer-guide/pullrequests#when-to-merge-a-pull-request)
```